addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -8,7 +8,7 @@
             <div class="row align-items-center justify-content-between">
                 <!-- Desktop Mode -->
                 <nav aria-label="breadcrumb" class="col-md-8 d-none d-md-flex">
-                    <ol class="breadcrumb bg-transparent mb-0 ps-0 py-0 overflow-hidden">
+                    <ol class="breadcrumb w-100 bg-transparent mb-0 ps-0 py-0 overflow-hidden">
```

Comment:
For the breadcrumb items: Does it make sens to limit the active breadcrumb item width to 25% (see: `w-25`)?

We can probably remove that class and add the class `flex-nowrap` on `.breadcrumb` to make sure that the breadcrumb does not span over multiple lines and add `flex-shrink-0` on the breadcrumb element we don't want to truncate. We will then utilize all the available space :smiley:

Reply:
Hello @amdi-odoo 

I was gonna create a task for the breadcrumb issue when I stumbled on this one ^^
I agree with @julien-banken, the `flex-nowrap` would be better imo. I Can't remember why we added the w-25 in the first place but it was a bad idea... You can have a look at the branch I worked on yesterday : https://github.com/odoo-dev/odoo/tree/16.0-website-slides-fix-truncated-breadcrumb-pko

Btw, you should retarget to v16 for this issue :wink:

Reply:
Hello @julien-banken @pko-odoo :wave:  :slightly_smiling_face: 

I created a new PR https://github.com/odoo/odoo/pull/165625  for the breadcrumbs issue.
I'm keeping this PR to fix the horizontal scrollbar as it needs to be fixed in 17.2 :slightly_smiling_face: 
Thanks for your branch @pko-odoo ! It was so perfect I didn't have anything to change :smile: I added you as co-author. Thanks!!

Thank you both for your help :slightly_smiling_face: